java/client/src/main/java/glide/api/commands/BaseCommands.java

```diff
@@ -7,19 +7,19 @@ public interface BaseCommands {
 
     /**
      * Executes a single command, without checking inputs. Every part of the command, including
-     * subcommands, should be added as a separate value in args.
+     * subcommands, should be added as a separate value in {@code args}.
      *
      * @remarks This function should only be used for single-response commands. Commands that don't
-     *     return response (such as SUBSCRIBE), or that return potentially more than a single response
```

Comment:
what is this change? is it related to this PR?
If this is required, please check these with a linter in the CI.

Reply:
Not related - will move to another PR.
This makes rendered javadoc a bit more readable. The linter only adds line breaks to keep lines up to 100 symbols.

Reply:
Updated in 1ea00600